includes/Modules/Sign_In_With_Google/Web_Tag.php

```diff
@@ -224,4 +245,4 @@ protected function render() {
 		BC_Functions::wp_print_inline_script_tag( $inline_script );
 		printf( "\n<!-- %s -->\n", esc_html__( 'End Sign in with Google button added by Site Kit', 'google-site-kit' ) );
 	}
-}
+}
```

Comment:
Let's restore the newline at the end of the file; not sure why that was removed 🤔